tests/config.py

```diff
@@ -45,7 +45,8 @@
 NODE_SIZE = os.getenv('NODE_SIZE', 'm1.medium')
 
 # The distro used on the underlying nodes
-DISTRO = os.getenv('DISTRO', 'SUSE')
+# Available options: openSUSE_k8s, SLES_CaaSP
+DISTRO = os.getenv('DISTRO', 'openSUSE_k8s')
```

Comment:
Isn't that now mixing the operating system distro with the k8s flavor?

Reply:
Yes. It's explained a little in the commit, but basically I think the base distro and deployment method are going to be tightly coupled.

For example, if we were to support Ubuntu, then after the bring up, installing the requirements for kubeadm will look different on each OS. eg we need to call apt instead of zypper and the exact packages may differ.

We're also unlikely to deploy vanilla/upstream kubernetes on SLES rather than CaaSP and vice-versa with openSUSE.

Reply:
oki